app/src/main/res/layout-land/appcoins_radio_button.xml

```diff
@@ -43,6 +44,105 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
         />
+
+    <androidx.constraintlayout.widget.Group
+        android:id="@+id/appc_balances_group"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:constraint_referenced_ids="appc_fiat_balance,appc_text_separator,appc_balance,balance_eth,eth_fiat_balance,eth_text_separator,eth_balance"
+        />
+
+    <TextView
+        android:id="@+id/appc_fiat_balance"
+        style="@style/TextAppearance.AppCompat.Body1.Roboto.Regular"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:fontFamily="sans-serif-medium"
+        android:lineSpacingExtra="4sp"
+        app:layout_constraintStart_toStartOf="@id/title"
+        app:layout_constraintTop_toBottomOf="@id/title"
+        tools:text="31.90 EUR"
+        />
+    <TextView
+        android:id="@+id/appc_text_separator"
+        style="@style/TextAppearance.AppCompat.Body1.Roboto.Medium"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="|"
```

Comment:
This attribute should be defined as a untranslatable string in the resources folder